test/testutils.js

```diff
@@ -110,6 +110,10 @@ function checkHeaders(res) {
     'X-Requested-With, Content-Type');
 }
 
+function dumpBody(res) {
+  console.log(res.body);
```

Comment:
This won't expand the object all the way, you'd need to use `JSON.stringify` or
`console.log(require('util').inspect(body, false, null));`